gradle-sls-packaging/src/main/java/com/palantir/gradle/dist/tasks/CreateManifestTask.java

```
@@ -154,6 +163,7 @@ final void createManifest() throws IOException {
                         .productVersion(getProjectVersion())
                         .putAllExtensions(getManifestExtensions().get())
                         .putExtensions("product-dependencies", productDependencies)
+                        .putExtensions("artifacts", artifacts)
```

as a safety mechanism, I think we should check to see if there is any `artifacts` extension currently, and then throw telling users to use the new thing instead of silently failing. As whatever they put before will be clobbered.